App/Controllers/TaskController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Controllers;
+
+use Core\View;
+use App\Models\User;
+use App\Models\Task;
+
+class TaskController {
+
+  public function index()
+  {
+      if(session_status() !== PHP_SESSION_ACTIVE)
+          session_start();
+
+      $sqlParams['teamleaderid']=$_SESSION["id"];
+      $sql = "SELECT * FROM TASKS WHERE USER_ID IS NULL UNION SELECT * FROM TASKS WHERE USER_ID IN (SELECT ID FROM USERS WHERE TEAM_LEADER_ID =:teamleaderid)";
```

Comment:
Move this logic inside Repository and keep Controllers only to return array of models :)